src/test/TestGhoBase.t.sol

```diff
@@ -175,43 +198,40 @@ contract TestGhoBase is Test, Constants, Events {
     STK_TOKEN = IStakedAaveV3(address(stkAaveProxy));
     USDC_TOKEN = new TestnetERC20('USD Coin', 'USDC', 6, FAUCET);
     USDC_4626_TOKEN = new MockERC4626('USD Coin 4626', '4626', address(USDC_TOKEN));
-    address ghoTokenAddress = address(GHO_TOKEN);
-    address discountToken = address(STK_TOKEN);
-    IPool iPool = IPool(address(POOL));
     WETH = new WETH9Mock('Wrapped Ether', 'WETH', FAUCET);
-    GHO_DEBT_TOKEN = new GhoVariableDebtToken(iPool);
-    GHO_STABLE_DEBT_TOKEN = new GhoStableDebtToken(iPool);
-    GHO_ATOKEN = new GhoAToken(iPool);
+    GHO_DEBT_TOKEN = new GhoVariableDebtToken(IPool(address(POOL)));
```

Comment:
using a mock instead of real Pool. is this not making other tests to fail?

Reply:
It was always using a mock. I made this change here because I needed to remove local variables because of stack too deep. I can actually add it back so it looks cleaner

Reply:
I changed it back to how it was before